Tavisca.Bootcamp.LanguageBasics.Exercise3/Program.cs

```diff
@@ -28,20 +29,143 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(int[] protein, int[] carbs, int[] fat, string[] dietPlans, int[] expected)
-        {
+        private static void Test(int[] protein, int[] carbs, int[] fat, string[] dietPlans, int[] expected)
+        {
             var result = SelectMeals(protein, carbs, fat, dietPlans).SequenceEqual(expected) ? "PASS" : "FAIL";
             Console.WriteLine($"Proteins = [{string.Join(", ", protein)}]");
             Console.WriteLine($"Carbs = [{string.Join(", ", carbs)}]");
-            Console.WriteLine($"Fats = [{string.Join(", ", fat)}]");
+            Console.WriteLine($"Fats = [{string.Join(", ", fat)}]");
             Console.WriteLine($"Diet plan = [{string.Join(", ", dietPlans)}]");
-            Console.WriteLine(result);
+            Console.WriteLine(result);
+        }
+
+        public static int[] SelectMeals(int[] protein, int[] carbs, int[] fat, string[] dietPlans)
+        {
+            var calorie = new int[protein.Length];
+            var dietList = new List<int>();
+            for(int i=0;i<protein.Length;i++)
+            {
+                calorie[i] = (protein[i] + carbs[i])*4 + fat[i]*9;
+            }
+            var index = new List<int>();
+            foreach (var item in dietPlans)
+            {
+                index.Clear();
+               if(string.IsNullOrWhiteSpace(item))
+                {
+                    index.Clear();
+                    index.Add(0);
+                }
+               else
+                {
+                    foreach (var ch in item)//CcPpFfTt

```

Comment:
Can you create a seperate method for this switch statement as the method is too big and not readable?